metadata-ingestion/cli-ingestion.md

```diff
@@ -1,66 +1,127 @@
 
-```shell
-# Requires Python 3.8+
+:::note Required Python Version
+Installing DataHub CLI requires Python 3.6+.
+:::
+
+<Tabs>
+<TabItem value="pypi" label="Install from pip" default>
+
+Run the following commands in your terminal:
+
+```
 python3 -m pip install --upgrade pip wheel setuptools
 python3 -m pip install --upgrade acryl-datahub
-# validate that the install was successful
-datahub version
-# If you see "command not found", try running this instead: python3 -m datahub version
+python3 -m datahub version
 ```
 
+Your command line should return the proper version of DataHub upon executing these commands successfully.
+
+</TabItem>
+</Tabs>
+
 Check out the [CLI Installation Guide](../docs/cli.md#installation) for more installation options and troubleshooting tips.
 
-After that, install the required plugin for the ingestion.
+
+## Installing Connector Plugins 
+
+Our CLI follows a plugin architecture. You must install connectors for different data sources individually. For a list of all supported data sources, see [the open source docs](metadata-ingestion/README.md#installing-plugins).
+Once you've found the connectors you care about, simply install them using `pip install`.
+For example, to install the `mysql` connector, you can run
 
 ```shell
-pip install 'acryl-datahub[datahub-rest]'  # install the required plugin
+pip install --upgrade acryl-datahub[mysql]
 ```
-
 Check out the [alternative installation options](../docs/cli.md#alternate-installation-options) for more reference.
 
 ## Configuring a Recipe
 
 Create a `recipe.yml` file that defines the source and sink for metadata, as shown below.
+[Recipes](metadata-ingestion/README.md#recipes) are yaml configuration files that serve as input to the Metadata Ingestion framework. Each recipe file define a single source to read from and a single destination to push the metadata.
+The two most important pieces of the file are the _source_ and _sink_ configuration blocks.
+
+The _source_ configuration block defines where to extract metadata from. This can be an OLTP database system, a data warehouse, or something as simple as a file. Each source has custom configuration depending on what is required to access metadata from the source. To see configurations required for each supported source, refer to the [Sources](metadata-ingestion/README.md#sources) documentation.
+
+The _sink_ configuration block defines where to push metadata into. Each sink type requires specific configurations, the details of which are detailed in the [Sinks](metadata-ingestion/README.md#sinks) documentation.
+In Acryl DataHub deployments, you _must_ use a sink of type `datahub-rest`, which simply means that metadata will be pushed to the REST endpoints exposed by your DataHub instance. The required configurations for this sink are
+
+1. **server**: the location of the REST API exposed by your instance of DataHub
+2. **token**: a unique API key used to authenticate requests to your instance's REST API
```

Comment:
let's move this down below - maybe a "using recipes with auth enabled" (don't actually like that title, but something like it)